redwood.toml

```diff
@@ -9,6 +9,7 @@
   bundler = "webpack"
   title = "Redwood App"
   port = 8910
+  host = "0.0.0.0"
```

Comment:
According to the [docs](https://redwoodjs.com/docs/app-configuration-redwood-toml#web), production uses `0.0.0.0` by default so this should not affect production. This was the main change to be able to hit the webserver from the host.

Reply:
Actually, these [docs](https://redwoodjs.com/docs/app-configuration-redwood-toml#running-in-a-container-or-vm) say to do this and for the api. I'll update this.